client/components/searchComponents/expandedSearchContent.tsx

```diff
@@ -33,14 +42,14 @@ export default function ExpandedSearchContent(props: {
                     },
                 }}
             >
-                {props.trackList.slice(0, 2).map((track, key) => (
+                {trackList.slice(0, 2).map((track, key) => (
                     <TrackCard
                         key={key}
                         track={track}
-                        updatePool={props.updatePool}
-                        token={props.token}
-                        disabled={props.disabled}
-                        enableAddButton={props.enableAddButton}
+                        updatePool={updatePool}
+                        token={token}
+                        disabled={disabled}
+                        enableAddButton={enableAddButton}
```

Comment:
Not for this ticket, but this snippet of code is repeated a lot in this file. A possible future improvement idea: make a component generic over content types for a single row of this, a list wrapper component and replace all these eight identical snippets with constructing that single wrapper component.

Reply:
I will make a new ticker for that